src/styles/constants/theme-print.scss

```
@@ -79,7 +80,8 @@
 		--table-row-hover-bg-color-rgb: var(--soft-gray-rgb);
 
 		&.print-portrait {
-			width: 900px;
+			width: 1200px;
+			// background-color: blue !important; // Switch to blue for debugging
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)